pkg/controller/controller_test.go

```diff
@@ -811,7 +814,9 @@ func (s *ControllerSuite) TestDeferPhase(c *C) {
 // 3. Phases have correct state in actionset status
 // 4. We don't render output artifacts if any of the phases failed
 func (s *ControllerSuite) TestDeferPhaseCoreErr(c *C) {
-	os.Setenv(kube.PodNSEnvVar, "test")
+	if err := os.Setenv(kube.PodNSEnvVar, "test"); err != nil {
```

Comment:
ditto.
Please fix all these cases.

Reply:
@julio-lopez  can i replicate above for every test file.